DFC.ServiceTaxonomy.Editor/DFC.ServiceTaxonomy.Editor.csproj

```diff
@@ -33,15 +33,20 @@
 
   <ItemGroup>
     <None Include="..\.editorconfig" Link=".editorconfig" />
+    <Content Remove="appsettings.Development_template.json" />
```

Comment:
Did you mean to leave this in?

Reply:
Yes, without it the file gets copied to the bin folder.